### PR TITLE
using docker config file for authentication

### PR DESCRIPTION
**Pull request by rbedemann**
this would also complete issue #181

### CODE REVIEW

File: src/main/java/com/spotify/docker/AbstractDockerMojo.java [hunk 1]
```diff
@@ -81,6 +81,9 @@
   @Parameter(property = "registryUrl")
   private String registryUrl;
 
+  @Parameter(property = "useConfigFile")
+  private String useConfigFile;
```
Comment (mattnworb) [inline src/main/java/com/spotify/docker/AbstractDockerMojo.java]: why not make this a `boolean`?

**Comment by mattnworb** [comment]
looks fine to me if the above comment is addressed

**Comment by mattnworb** [comment]
and thanks for the contribution!

File: src/main/java/com/spotify/docker/AbstractDockerMojo.java (hunk 1 above)
Comment (rbedemann) [inline src/main/java/com/spotify/docker/AbstractDockerMojo.java]: makes sense ;) second commit now solves this issue

**Comment by mattnworb** [comment]
👍

**Pull request merged**